### ISSUE TITLE
Add multiple version support in caelus scripts

### ISSUE THREAD

**Issue by cnsidero** [sayerhs/cpl#16]
CPL can handled multiple versions but that functionality is only accessible through the CPL module in a custom Python script. Running multiple versions through the `caelus run` or `caelus tasks` should be supported with, for example, a cmd option:

caelus  --cml-version="8.04-dev" run

where "8.04-dev" would be a version specified in the configuration file.

**Comment by sayerhs** [comment]
Feature implemented in commit df8c77b6e19. Example usage:

```
caelus --cml-version 6.10 run pimpleSolver

caelus_tutorials --cml-version 8.04 -f run_validation.yaml
```

**Comment by sayerhs** [comment]
An alternate approach is to create a `caelus.yaml` configuration file in the run directory and with the following contents:

```
caelus:
  caelus_cml:
    default: "6.10"    # Change version here before invoking CPL commands
```

**Comment by sayerhs** [comment]
Implemented in 0b204da6131bfd
